src/luawrap.cpp

```diff
@@ -27,17 +27,23 @@ void LuaReportErrors(lua_State *L, int status)
     }
 }
 
+void LuaRunString(lua_State *L, const char *buffer)
+{
+    LuaReportErrors( L, luaL_loadstring(L, buffer) || lua_pcall(L, 0, 0, 0) );
+}
+
 void LuaRunFile(lua_State *L, const char *filename)
 {
-    char *buffer;
+    char *buffer; // TODO(Rasmus): Replace with RAII
     PhysFSLoadFile(filename, buffer);
```

Comment:
All file access goes through [PhysicsFS](https://icculus.org/physfs/) to make it easy to pack files into archives. It also acts like a sandbox/jail.